superuser.ts

```diff
@@ -194,8 +195,30 @@ export function initializeSuperuserEndpoints(state: State) {
         return;
       }
 
-      const initialScene = initialSceneID ?
-        await state.scenes.findOne({ "_id": initialSceneID }) : null;
+      let initialScene: WithId<MongoScene> | null = null;
+      if (initialSceneID === null) {
+        // Basic implementation for now:
+        // If no initial ID is given in the request, then take the
+        // either the first featured scene for the given day, or the first scene
+        // in the feature queue if there are no features that day.
+        // We can make this better once we have the scheduler set up
+        //
+        // NB: Once the scheduler is set up, we can avoid any sort of
+        // fiddly date logic by just passing in the ID of the desired
+        // scene in the request (in the scheduled job)
+        const features = await getFeaturesForDate(state, new Date());
+        const firstFeature = await features.next();
+        if (firstFeature !== null) {
+          initialSceneID = firstFeature.scene_id;
+        } else {
+          const result = await tryPopFromFeatureQueue(state);
+          const queueDoc = result.value;
+          initialSceneID = queueDoc?.scene_ids[0] ?? null;
+        }
```

Comment:
I feel like this logic would be better located in a little function in `features.ts`, so that this code can just write something like

```
initialSceneID = await getCurrentFeaturedScene(state);
```

or something like that. The other file seems like a better place to put that kind of logic, which depends on relatively detailed properties of the feature system.

Reply:
Reupping this comment.